src/LibraryManager.Vsix/Contracts/PerProjectLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Vsix.Contracts
+{
+    internal class PerProjectLogger : ILogger
+    {
+        private string _projectName;
+
+        public PerProjectLogger(string projectName)
+        {
+            _projectName = projectName;
+        }
+
+        public void Log(string message, LogLevel level)
+        {
+            Logger.Log($"{message} ({_projectName})", level);
```

Comment:
_projectName can be null here if we fail to get projectName

Reply:
Should we have a fallback string for a default if we can't resolve a project name?

Reply:
Not sure what the default project name would be. I think if it's null, we should just omit it. Anything else would look bad to the users. I'm not sure in which case project name would be null though - do you guys know of such cases? 

Anyway - I added null check and will be using empty string for now if there is no project name - so the output would be as before this change.